.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cub"]
```

Comment:
@doctorcolinsmith @stanleytsang-amd Is this acceptable?

Reply:
Should the submodule point to our hipCUB instead of NVIDIA's cub?

Reply:
I've checked with Istvan and our cmake script we've developed for rocThrust does not directly support the CUDA backend (cmake will throw an error message if nvcc is selected). The decision for this was made at the initial stages of development, where it was agreed that users looking to use CUDA could revert to the official repo (especially given the official repo was versions ahead).

I'm not sure what your CI does with cub after cloning it (the solution was based on your CI), but perhaps after this is merged, we could discuss the best way around this.

Reply:
Makes sense to me. CUDA compatibility can be a separate discussion.